.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on: pull_request
+jobs:
+  test:
+    name: Test -- ${{ matrix.testenv }}
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        testenv:
+          - serverless
```

Comment:
Not much in the matrix, however it might be nice to have other options ready at some point (eg. one without Flask!).